### ISSUE TITLE
Can we pass eglGetProcAddress pointer to cardboard?

### ISSUE THREAD

**Issue by phuang** [googlevr/cardboard#194]
Seems cardboard is linked with GL libraries in build time, right? Could we support passing an getProcAddress function pointer to cardboard to force it to use an alternative GL implementation in runtime?

**Comment by jballoffet** [comment]
Exactly, the Cardboard SDK is linked against the OpenGL libraries in build time (per https://github.com/googlevr/cardboard/blob/4382d959a62196405ab75a237b5c39094a137c6e/sdk/CMakeLists.txt#L88). Currently, it does not support using an alternative GL implementation by setting it on runtime, nevertheless, we would like to know what would be your use case to better define and prioritize this feature request and maybe give some pointers about a possible implementation.
